### ISSUE TITLE
VideoPress: Make videos and quick actions accessible via keyboard

### ISSUE THREAD

**Issue by dhasilva** [Automattic/jetpack#27331]
In the grid view, videos are not focusable via keyboard, and on the list view the actions are only shown on hover.

p1668009383019279/1668003224.122799-slack-C03TA48NSUX
